types/foundry-vtt-api/types/framework/entity.d.ts

```diff
@@ -0,0 +1,562 @@
+declare interface BaseEntityData {
+    name: string;
+    type: string;
+    flags: any;
+    data: any;
+}
+
+/**
+ * An abstract class pattern for all primary data entities within the Foundry VTT Framework
+ * An entity represents a primary data concept, for example: Actor, Item, Scene, or ChatMessage.
+ * Employing this abstraction layer ensures similar behavior and workflow for all entity types.
+ *
+ * Documentation for this class is provided for reference, but developers should not extend this class directly,
+ * instead work with or extend the Entity implementations that are referenced in this section of the API documentation.
+ *
+ * Entities are instantiated by providing their base data, and an optional Array of Application instances which should
+ * be automatically refreshed when the Entity experiences an update.
+ *
+ * @param data		The data Object with which to create the Entity
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/framework/entity.d.ts;
  Line: 19;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.